Examples/TicTacToe/TicTacToeApp.js

```diff
@@ -236,6 +238,29 @@ var TicTacToeApp = React.createClass({
   }
 });
 
+var TITLE_SIZE = Platform.isTVOS ? 117 : 39;
+var TITLE_FONT = Platform.isTVOS ? 'Helvetica' : 'Chalkduster';
+var TITLE_MARGIN_BOTTOM = Platform.isTVOS ? 60 : 20;
+
+var BOARD_BORDER_RADIUS = Platform.isTVOS ? 30 : 10;
+var BOARD_PADDING = Platform.isTVOS ? 15 : 5;
+
+var CELL_BORDER_RADIUS = Platform.isTVOS ? 15 : 5;
+var CELL_MARGIN_SIZE = Platform.isTVOS ? 15 : 5;
+
+var CELL_SIZE = Platform.isTVOS ? 240 : 80;
+var CELL_FONT = Platform.isTVOS ? 'Helvetica-Bold' : 'AvenirNext-Bold';
+var CELL_FONT_SIZE = Platform.isTVOS ? 150 : 50;
+
+var OVERLAY_FONT = Platform.isTVOS ? 'Helvetica-Bold' : 'AvenirNext-DemiBold';
+var OVERLAY_FONT_SIZE = Platform.isTVOS ? 120: 40;
+var OVERLAY_MARGIN_SIZE = Platform.isTVOS ? 60 : 20;
+
+var NEW_GAME_FONT = Platform.isTVOS ? 'Helvetica-Bold' : 'AvenirNext-DemiBold';
+var NEW_GAME_FONT_SIZE = Platform.isTVOS ? 60 : 20;
+var NEW_GAME_PADDING_SIZE = Platform.isTVOS ? 60 : 20;
+var NEW_GAME_BORDER_RADIUS = Platform.isTVOS ? 15 : 5;
```

Comment:
Might be easier to inline all of these conditions directly into the styles below? That way there's no need to add the extra lines.